File: bukkit/elysium-characters-bukkit/src/main/java/com/seventh_root/elysium/characters/bukkit/race/BukkitRace.java
package com.seventh_root.elysium.characters.bukkit.race;

import com.seventh_root.elysium.api.character.Race;

public class BukkitRace implements Race {

    private int id;
    private final String name;

    public BukkitRace(int id, String name) {
        this.id = id;
        this.name = name;
    }

    public BukkitRace(String name) {
        this.name = name;
    }

    @Override
    public int getId() {
        return id;
    }

    @Override
    public void setId(int id) {
        this.id = id;
    }

    @Override
    public String getName() {
        return name;
    }

}

File: bukkit/elysium-characters-bukkit/src/main/kotlin/com/seventh_root/elysium/characters/bukkit/race/BukkitRace.kt
package com.seventh_root.elysium.characters.bukkit.race

import com.seventh_root.elysium.characters.bukkit.race.Race

class BukkitRace : Race {

    override var id: Int = 0
    override val name: String

    constructor(id: Int, name: String) {
        this.id = id
        this.name = name
    }

    constructor(name: String) {
        this.name = name
    }

}
